languages/english.py

```diff
@@ -169,6 +173,11 @@ def t_OBSTACLERIGHT(t):
     t.value = 'OBSTACLERIGHT'
     return t
 
+def t_COINS(t):
```

Comment:
What's the point of this rule?

Reply:
We wanted to support n=number of coins na instead of a function call..So I have added this